### PR TITLE
docs(event-handling): review src/guide/essentials/event-handling.md

### PR DESCRIPTION
**Pull request by coder-xiaomo**
完成校对 #65

### CODE REVIEW

**Review by watonyweng — APPROVED** [review]

File: src/guide/essentials/event-handling.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 ## 监听事件 {#listening-to-events}
 
-你可以使用 `v-on` 指令 (简写为 `@`) 来监听 DOM 事件和触发 JavaScript 代码运行。用法可以是 `v-on:click="methodName"` 或简写为 `@click="handler"`。
+你可以使用 `v-on` 指令 (简写为 `@`) 来监听 DOM 事件和运行 JavaScript 代码。用法： `v-on:click="methodName"` 或 `@click="handler"` 。
```
Comment (veaba) [inline src/guide/essentials/event-handling.md:5]:
```suggestion
你可以使用 `v-on` 指令 (简写为 `@`) 来监听 DOM 事件和运行 JavaScript 代码。用法：`v-on:click="methodName"` 或 `@click="handler"`。
```

File: src/guide/essentials/event-handling.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 
-事件处理器的值应该是下面两种其中之一：
+事件处理器的值可以是：
 
-1. **内联事件处理器：**事件被触发时执行的内联 JavaScript 语句 (与原生的 `onclick` attribute 类似)。
+1. **内联事件处理器：** 事件被触发时执行的内联 JavaScript 语句 (与 `onclick` 类似) 。
```
Comment (veaba) [inline src/guide/essentials/event-handling.md:9]:
```suggestion
1. **内联事件处理器：**事件被触发时执行的内联 JavaScript 语句 (与 `onclick` 类似) 。
```

File: src/guide/essentials/event-handling.md [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
 
-2. **用方法作为事件处理器：**一个组件的属性名、或对某个方法的访问 (比如 `@click="some.method"`)。
+2. **方法事件处理器：** 一个组件的属性名、或对某个方法的访问。
```
Comment (veaba) [inline src/guide/essentials/event-handling.md:11]:
```suggestion
2. **方法事件处理器：**一个组件的属性名、或对某个方法的访问。
```

**Review by veaba — APPROVED** [review]
format space

**Pull request merged**